pandas/_libs/tslibs/np_datetime.pyx

```diff
@@ -194,6 +194,11 @@ cdef get_implementation_bounds(
         raise NotImplementedError(reso)
 
 
+cdef object dts_to_iso_string(npy_datetimestruct *dts):
```

Comment:
should the return here be str?

Reply:
No reason - didn't think much of it. Makes sense though - just pushed this up